src/net_processing.cpp

```diff
@@ -1975,7 +1975,8 @@ void PeerManagerImpl::ActiveTipChange(const CBlockIndex& new_tip, bool is_ibd)
 void PeerManagerImpl::BlockConnected(
     ChainstateRole role,
     const std::shared_ptr<const CBlock>& pblock,
-    const CBlockIndex* pindex)
+    const CBlockIndex* pindex,
+    bool is_ibd)
```

Comment:
there should be a better way to do this than passing around a boolean over so many layers. I'll review it properly this week.